docs/account.mdx

```diff
@@ -0,0 +1,630 @@
+Unlock your account.
+? Passphrase:
+You are about to sign the following transaction:
+Format: plain
+Method: consensus.Deposit
+Body:
+  To: oasis1qpupfu7e2n6pkezeaw0yhj8mcem8anj64ytrayne
+  Amount: 10.0 TEST
+Authorized signer(s):
+  1. W4Xii5x0oaOlDa1wVAa1VTmp18NRt4Z6NseaoDtRA7o= (ed25519)
+     Nonce: 0
+Fee:
+  Amount: 0.0 TEST
+  Gas limit: 11310
+  (gas price: 0.0 TEST per gas unit)
+
+Network:  testnet
+ParaTime: sapphire
+Account:  oscar
+```
+
+## Withdraw
+
+`account withdraw <amount> [to]` will withdraw funds from your ParaTime account
+to a consensus address.
+
+```
+$ oasis account withdraw 10 oscar
+Unlock your account.
+? Passphrase:
+You are about to sign the following transaction:
+Format: plain
+Method: consensus.Withdraw
+Body:
+  To: oscar (oasis1qq59js97ytn2xcnscyx9n25z5y4dc6z6qcqsydyr)
+  Amount: 10.0 TEST
+Authorized signer(s):
+  1. ArEjDxsPfDvfeLlity4mjGzy8E/nI4umiC8vYQh+eh/c (secp256k1eth)
+     Nonce: 0
+Fee:
+  Amount: 0.0011317 TEST
+  Gas limit: 11317
+  (gas price: 0.0000001 TEST per gas unit)
+
+Network:  testnet
+ParaTime: sapphire
+Account:  eugene
+```
+
+If address is not provided, the address of the account inside ParaTime will be
+used as a consensus address. You cannot use the address of your `secp256k1`
+account, because this signature scheme is not supported on consensus.
+
+Withdrawal transactions are not free of charge and the fee will be deducted
+**from your ParaTime balance**.
+
+You can specify a custom Oasis address which you want to withdraw your ROSE to by appending it to the withdraw command:
+
+```
+$ oasis accounts withdraw 10 oasis1qpxhsf7xnm007csw2acaa7mta2krzpwex5c90qu6
+Unlock your account.
+? Passphrase:
+You are about to sign the following transaction:
+Format: plain
+Method: consensus.Withdraw
+Body:
+  To: webext (oasis1qpxhsf7xnm007csw2acaa7mta2krzpwex5c90qu6)
+  Amount: 10.0 TEST
+Authorized signer(s):
+  1. ArEjDxsPfDvfeLlity4mjGzy8E/nI4umiC8vYQh+eh/c (secp256k1eth)
+     Nonce: 0
+Fee:
+  Amount: 0.0011317 TEST
+  Gas limit: 11317
+  (gas price: 0.0000001 TEST per gas unit)
+
+Network:  testnet
+ParaTime: sapphire
+Account:  eugene
+```
+
+## Delegate and Undelegate stake
+
+To stake your tokens on consensus layer, run `account delegate <amount> <to>`.
+This will delegate the specified amount of tokens to a validator.
+
+```
+$ oasis account delegate 20 oasis1qpkl3vykn9mf4xcq9eevmey4ffrzf0ajtcpvd7sk --account oscar --no-paratime
+Unlock your account.
+? Passphrase:
+You are about to sign the following transaction:
+Method: staking.AddEscrow
+Body:
+  To:     oasis1qpkl3vykn9mf4xcq9eevmey4ffrzf0ajtcpvd7sk
+  Amount: 20.0 TEST
+Nonce:  7
+Fee:
+  Amount: 0.0 TEST
+  Gas limit: 1271
+  (gas price: 0.0 TEST per gas unit)
+
+Network:  testnet
+ParaTime: none (consensus layer)
+Account:  oscar
+```
+
+Once your tokens are staked, they are denoted as *shares* since the number of
+tokens will increase over time due to positive interest rate, while the number
```

Comment:
This needs to be reworded to say change as it may also be reduced in case of slashing etc. Also this probably shouldn't say "interest rate" as it is based on the current staking reward schedule, commission rate, etc.